content/rc/cloud-integrations/aws-marketplace/aws-bedrock.md

```diff
@@ -0,0 +1,107 @@
+
+Amazon Bedrock allows you to choose Redis Cloud as the [vector database](https://redis.com/solutions/use-cases/vector-database/) for your models. Once your database is set up and connected to Amazon Bedrock, it will import text data from an Amazon Simple Storage Service (S3) bucket into Redis Cloud and use it to extract relevant information when prompted.
+
+For more information about the Redis integration with Amazon Bedrock, see the [Amazon Bedrock integration blog post](https://redis.com/blog/amazon-bedrock-integration-with-redis-enterprise/).
+
+## Set up Redis for Bedrock
+
+You need to set up your Redis Cloud database before you can set it as the vector database in Amazon Bedrock. To do this, you need to:
+
+- [Create a subscription and database in Redis Cloud](#create-subscription)
+- [Enable Transport Layer Security (TLS) for the database and save the certificates](#get-certs)
+- [Store database credentials in AWS secrets manager](#store-secret)
+- [Create a vector index in your database](#create-vector-index) for Bedrock to use
+
+Once you have set up the database, you can use the information
+
+### Create a subscription and database {#create-subscription}
+
+To set up a Redis Cloud instance for Bedrock:
+
+1. If you're new to Redis Cloud, sign up for Redis Cloud using the [AWS Marketplace integration]({{<relref "/rc/cloud-integrations/aws-marketplace/">}}). 
+
+1. [Create a flexible subscription]({{<relref "/rc/subscriptions/create-flexible-subscription">}}). In the **Sizing** tab, when creating your first database, make sure to have the following settings:
+
+    - Add the **RediSearch** and **RedisJSON** advanced capabilities to your database.
+    - Set the size of the database based on the amount of data that will be pulled from your Simple Storage Service (S3) [bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-buckets-s3.html). See [Find out the size of your S3 buckets](https://aws.amazon.com/blogs/storage/find-out-the-size-of-your-amazon-s3-buckets/) to find out how much training data is stored in your S3 bucket and pick the closest size from the table below. 
+
+    | Total Size of Documents in S3 | Database size without replication | Database size with replication |
+    |-------------------------------|-----------------------------------|--------------------------------|
+    | 10,000 kb                     | 135 Mb                             | 270 Mb                         |
+    | 100,000 kb                    | 1.35 Gb                            | 2.7 Gb                         |
+    | 1,000,000 kb                  | 13.5 Gb                              | 27 Gb                          |
+    | 10,000,000 kb                 | 135 Gb                             | 270 Gb                         |
+
+    For more information on sizing, see the [Bedrock integration blog post](https://redis.com/blog/amazon-bedrock-integration-with-redis-enterprise#right-size-your-database-for-amazon-bedrock).
```

Comment:
Another 404

Reply:
Yup, the 404s are future links. I'll check them again before I publish.